src/create-element.js

```diff
@@ -62,6 +62,7 @@ export function createVNode(type, props, text, key, ref) {
 		_lastDomChild: null,
 		_component: null
 	};
+	vnode._ = vnode;
```

Comment:
Could this be given a more self-documenting name, which is then shortened in the build process like the other private properties are?

Reply:
Is this better, it's _self now. I must admit the name was a bit odd of me to do, I wanted to contain bundle size increase but you're right that's what we have our mangle file for :D thanks for pointing it out